README.md

```diff
@@ -4,10 +4,10 @@ Secretary service of Stay Safe project
 
 ## Run Flask API
 If you haven't build yet:
-<pre><code>$ docker-compose up --build</code></pre>
+<pre><code>$ docker-compose up web db --build</code></pre>
```

Comment:
this command is not working:
![image](https://user-images.githubusercontent.com/30883179/94238986-7f029e80-fee7-11ea-8234-34904cfbd858.png)

```suggestion
<pre><code>$ docker-compose build web db </code></pre>
```